.github/workflows/merge.yml

```diff
@@ -234,6 +234,12 @@ jobs:
           SN_LOG: "all"
         timeout-minutes: 5
 
+      - name: Start a client to estimate a files upload cost
+        run: cargo run --bin safe --release -- --log-output-dest=data-dir files estimate "./resources"
```

Comment:
I took out the "retry-strategy quick" bit from here. 
I don't know what it does, but now it passes the tests.

Reply:
The nature of the error ::
```
PS C:\Users\lol_h\RustroverProjects\safe_network> cargo run --bin safenode-manager --features local-discovery -- run --build
   Compiling sn_transfers v0.15.5 (C:\Users\lol_h\RustroverProjects\safe_network\sn_transfers)
   Compiling sn_protocol v0.14.1 (C:\Users\lol_h\RustroverProjects\safe_network\sn_protocol)
   Compiling sn_networking v0.13.19 (C:\Users\lol_h\RustroverProjects\safe_network\sn_networking)
   Compiling sn_client v0.104.14 (C:\Users\lol_h\RustroverProjects\safe_network\sn_client)
   Compiling sn_node v0.104.22 (C:\Users\lol_h\RustroverProjects\safe_network\sn_node)
   Compiling sn_node_rpc_client v0.4.54 (C:\Users\lol_h\RustroverProjects\safe_network\sn_node_rpc_client)
   Compiling sn-node-manager v0.3.1 (C:\Users\lol_h\RustroverProjects\safe_network\sn_node_manager)
    Finished dev [unoptimized + debuginfo] target(s) in 57.11s
     Running `target\debug\safenode-manager.exe run --build`
Error:
   0: missing field `local` at line 1 column 829

Location:
   sn_protocol\src\node_registry.rs:182
```